src/test/java/model/CarTest.java

```diff
@@ -0,0 +1,84 @@
+package model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Nested;
+import view.InputView;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import java.util.List;
+
+class CarTest {
+
+    private String input;
+
+    private static final int POSITION_THRESHOLD_FOR_MOVE = 4;
+    private static final int POSITION_THRESHOLD_FOR_NO_MOVE = 3;
+
+    @BeforeEach
+    void setUp() {
+        input = "neo,brie,brown";
+    }
+
+    @Nested
+    @DisplayName("자동차 이름 관련 테스트")
+    class CarNameTests {
+
+        @Test
+        @DisplayName("자동차 리스트 입력 시 쉼표로 자동차를 분리하는 지 검증한다")
+        void testSplitCarName() {
+            List<String> carNames = InputView.parseCarNames(input);
+
+            assertThat(carNames).hasSize(3);
+        }
+
+        @Test
+        @DisplayName("자동차 이름이 5자 이상일 경우 예외가 발생하는 지 검증한다")
+        void testCarNameLengthTooLong() {
+            assertThatThrownBy(() -> new Car("SuperCar"))
+                    .isInstanceOf(IllegalArgumentException.class);
+        }
+
+        @Test
+        @DisplayName("자동차 이름이 5자 이하일 경우 자동차 객체가 정상 생성되는 지 검증한다")
+        void testCarNameLengthValid() {
+            Car car1 = new Car("Car1");
+            assertThat(car1.getCarName()).isEqualTo("Car1");
+        }
+
+        @Test
+        @DisplayName("자동차 리스트가 null일 경우 예외가 발생하는 지 검증한다")
+        void testGetCarName() {
+            String input = null;
+
+            assertThatThrownBy(() -> {
+                InputView.parseCarNames(input);
+            }).isInstanceOf(IllegalArgumentException.class)
+                    .hasMessageContaining("자동차 리스트가 null일 수 없습니다.");

```

Comment:
```suggestion
            assertThatThrownBy(() -> InputView.parseCarNames(input))
                    .isInstanceOf(IllegalArgumentException.class)
                    .hasMessageContaining("자동차 리스트가 null일 수 없습니다.");
```

람다 표현식으로 대체할 수 있을 것 같네요! 😁

Reply:
넵 람다표현식에 더 익숙해질 수 있도록 하겠습니다 !
말씀해주신대로 람다 표현식으로 리펙토링 하였습니다 감사합니다!